app/views/interactions/_interaction_summary_tab.html.haml

```diff
@@ -0,0 +1,45 @@
+/ special partial for summary tab of interaction view
+
+/ displays main interaction info, other info, and publications
+
+/ - card layout, just like gene drug summary tab
+/ - main info in one column? then same 2nd column as gene drug summary tab? so main info replaces aliases?
+/ - 
+
+%div(class="item-content")
+	%div{class: "cell category", style: "float: left; width: 50%" }
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.